.ts-setup.el

```diff
@@ -6,9 +6,8 @@
          (javascript . ("https://github.com/tree-sitter/tree-sitter-javascript" "v0.20.1" "src"))
          (json . ("https://github.com/tree-sitter/tree-sitter-json" "v0.20.2"))
          (go . ("https://github.com/tree-sitter/tree-sitter-go" "v0.20.0"))
-         (ocaml . ("https://github.com/tree-sitter/tree-sitter-ocaml" "v0.24.2" "grammars/ocaml/src"))
-         (ocaml_interface . ("https://github.com/tree-sitter/tree-sitter-ocaml" "v0.24.2" "grammars/interface/src"))
-         (ocaml_type . ("https://github.com/tree-sitter/tree-sitter-ocaml" "v0.24.2" "grammars/type/src"))
+         (ocaml . ("https://github.com/ccasin/tree-sitter-ocaml" "jane-street-extensions" "grammars/ocaml/src"))
+         (ocaml_interface . ("https://github.com/ccasin/tree-sitter-ocaml" "jane-street-extensions" "grammars/interface/src"))
```

Comment:
Remove the commit be978d6d0c5f48347c54800244ed8a461c9f73ff 
We don't want to add the oxcaml grammars on this branch.